sdk/appconfiguration/azure-data-appconfiguration/swagger/src/main/java/AppConfigCustomization.java

```diff
@@ -0,0 +1,146 @@
+            .setDescription("Populates the {@link ConfigurationSetting#getLabel()} from the service.");
+
+        classCustomization.getConstant("VALUE")
+            .getJavadoc()
+            .setDescription("Populates the {@link ConfigurationSetting#getValue()} from the service.");
+
+        classCustomization.getConstant("CONTENT_TYPE")
+            .getJavadoc()
+            .setDescription("Populates the {@link ConfigurationSetting#getContentType()} from the service.");
+
+        classCustomization.getConstant("LAST_MODIFIED")
+            .getJavadoc()
+            .setDescription("Populates the {@link ConfigurationSetting#getLastModified()} from the service.");
+
+        classCustomization.getConstant("TAGS")
+            .getJavadoc()
+            .setDescription("Populates the {@link ConfigurationSetting#getTags()} from the service.");
+
+        classCustomization.getConstant("ETAG")
+            .getJavadoc()
+            .setDescription("Populates the {@link ConfigurationSetting#getETag()} from the service.");
+
+        MethodCustomization fromString = classCustomization.getMethod("fromString");
+        fromString.getJavadoc()
+            .setDescription("Creates or finds a {@link SettingFields} from its string representation.")
+            .setReturn("the corresponding {@link SettingFields}");
+        fromString.removeAnnotation("JsonCreator");
+
+        classCustomization = classCustomization.customizeAst(
+            ast -> ast.getClassByName("KeyValueFields")
+                       .get()
+                       .getDefaultConstructor().get()
+                       .remove());
+
+        // Rename LOCKED to IS_READ_ONLY
+        classCustomization = classCustomization.renameEnumMember("LOCKED", "IS_READ_ONLY");
+        classCustomization.getConstant("IS_READ_ONLY")
+            .getJavadoc()
+            .setDescription("Populates the {@link ConfigurationSetting#isReadOnly()} from the service.");
+
+        // Change class name to SettingFields
+        ClassCustomization settingFields = classCustomization.rename("SettingFields");
+        settingFields.getJavadoc()
+            .setDescription(joinWithNewline(
+                "",
+                "Fields in {@link ConfigurationSetting} that can be returned from GET queries.",
+                "",
+                "@see SettingSelector",
+                "@see ConfigurationAsyncClient",
+                ""
+            ));
+
+        // Add toStringMapper static new method to SettingFields
+        addToStringMapper(settingFields);
+    }
+
+    private ClassCustomization addToStringMapper(ClassCustomization classCustomization) {
+        return classCustomization.customizeAst(ast -> {
+            String className = classCustomization.getClassName();
+            ClassOrInterfaceDeclaration clazz = ast.getClassByName(className).get();
+
+            clazz.addMethod("toStringMapper", Modifier.Keyword.STATIC, Modifier.Keyword.PUBLIC).setType("String")
```

Comment:
Let's also mark this method as deprecated as we shouldn't be using it anymore. It seems its purpose was to fix the fact we had the wrong casing for the enums.